src/main/resources/logback.xml

```diff
@@ -2,28 +2,28 @@
 <!DOCTYPE logback>
 <configuration>
 
-   <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
-      <filter class="ch.qos.logback.classic.filter.LevelFilter">
-         <level>INFO</level>
-         <onMatch>ACCEPT</onMatch>
-         <onMismatch>DENY</onMismatch>
-      </filter>
-      <encoder>
-         <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n</pattern>
-      </encoder>
-   </appender>
-
-   <appender name="FILE" class="ch.qos.logback.core.FileAppender">
-      <file>/var/log/scalar/cassy.log</file>
-      <append>true</append>
-      <encoder>
-         <pattern>%d{yyyy-MM-dd} %d{HH:mm:ss.SSS} [%thread] %-5level %logger{35} - %msg%n</pattern>
-      </encoder>
-   </appender>
-
-   <root level="info">
-      <appender-ref ref="FILE" />
-      <appender-ref ref="STDOUT" />
-   </root>
+    <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+        <filter class="ch.qos.logback.classic.filter.LevelFilter">
+            <level>INFO</level>
+            <onMatch>ACCEPT</onMatch>
+            <onMismatch>DENY</onMismatch>
+        </filter>
+        <encoder>
+            <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n</pattern>
+        </encoder>
+    </appender>
 
+    <appender name="FILE" class="ch.qos.logback.core.FileAppender">
+        <file>/var/log/scalar/cassy.log</file>
+        <append>true</append>
+        <encoder>
+            <pattern>%d{yyyy-MM-dd} %d{HH:mm:ss.SSS} [%thread] %-5level %logger{35} - %msg%n</pattern>
+        </encoder>
+    </appender>
+    <logger name="net.schmizz.sshj" level="WARN"/>
+    <logger name="com.hierynomus.sshj" level="WARN"/>
```

Comment:
This 2 libraries print non informative logs in the INFO level when using giraffe file copy methods so I added these filters to hide them.